R/dependencies.R

```diff
@@ -90,6 +91,7 @@ checkDependencies <- function(dep_path,
 #'
 #' @return Character vector of any misaligned package versions between
 #' rplatform \code{dependencies.yaml} and package \code{DESCRIPTION}.
+#' @keywords internal
```

Comment:
We want to hide this function from public view? 👀

Reply:
Yes. It's for our internal purposes only.